src/kernels/ipywidgets-message-coordination/scriptUriConverter.ts

```diff
@@ -19,11 +19,11 @@ export class ScriptUriConverter implements ILocalResourceUriConverter {
         return this.requestUriEmitter.event;
     }
     public get rootScriptFolder(): Uri {
-        return Uri.file(this._rootScriptFolder);
+        return this._rootScriptFolder;
```

Comment:
This file isn't node specific anymore, but doesn't actually work in the web because we can't write to the extensionUri location. It isn't used for web right now but once [#10060](https://github.com/microsoft/vscode-jupyter/issues/10060) is implemented, it can be used to support that scenario. 

That's why I left this being usable for web too.